docs/en/reference/migrating.md

```diff
@@ -0,0 +1,41 @@
+# Migrating to a New Computer
+
+If you're taking full advantage of Zettlr's functionality, you'll eventually create an extensive library of workspaces, interconnected notes, and likely a few custom settings, too. When the time comes to replace your computer, or perhaps migrate to a new operating system, you'll obviously want to take this data with you.
+
+You'll be pleased to know that this is a straightforward process, which typically involves three steps:
+
+1. Copying the workspaces containing your notes to the new machine.
+2. Copying your application settings to the new machine.
+3. Reconnecting your reference manager (if you use one).
+
+## Copying the Workspaces Containing Your Notes to the New Machine
+
+Because Zettlr workspaces are simply folders on your computer and your notes are simply text files, moving this data to a new computer is as simple as copying these folders from one computer to the next. All of the metadata about your notes and the connections between them — note IDs, tags, etc. — is saved to the individual note files themselves, meaning this information travels right along with the files.
+
+Once you've copied the workspace folders containing your notes from your old computer and placed them on the new machine, you can add each workspace and its files to your new Zettlr install simply by going to the `File > Open Workspace …` menu option and selecting that workspace folder. Note that this is the same process you would use you if were adding any other folder to Zettlr. The workspace folder, including the notes it contains, will immediately be populated into your new Zettlr install. The connections between notes will also be retained, though if you have links to notes that live in different workspaces, you'll have to make sure you've likewise opened those workspace folders and their contents within your new Zettler install.
```

Comment:
I would argue that we shouldn't state this here because one would need to do that after copying the config file. Right now, some people may be inclined to do a fresh install, open their folders, then copy the config and be confused why their folders are gone again.

Maybe it might make sense to just end the "move files" section and continue with the configuration here.

Reply:
I get what you're suggesting, but have a couple questions. Per our previous discussion, I removed the advice to copy `config.json` to the new machine. If the user refrains from copying this file over, is this still an issue? If not, would it be more straightforward to just have an aside about what happens if you _were_ to copy over `config.json`, so users who do so aren't confused, but to otherwise assume they won't be migrating this file?

Reply:
I remember that I was like "Oh no don't copy the config" BUT then I realized that (a) there are the autocorrect keys stored in the config (not the best design I know) and (b) the config should automatically just remove all the links that are no longer valid.

So, bottom line is: Copying over the config should actually be fine.

Reply:
I added in guidance on copying `config.json` and reworked/reflowed the other guidance as necessary.